builder/ebsvolume/builder.go

```diff
@@ -161,6 +161,13 @@ func (b *Builder) Prepare(raws ...interface{}) ([]string, []string, error) {
 			"Packer, inclusion of enable_t2_unlimited will error your builds.")
 	}
 
+	for _, configVolumeMapping := range b.config.VolumeMappings {
+		if configVolumeMapping.SnapshotDescription != "" && configVolumeMapping.SnapshotVolume != true {
+			warns = append(warns, "snapshot_description is ignored when "+
```

Comment:
Yeah I think it's good enough, if we're able to pinpoint which volume is the problem that would be even better, but not sure there's something we can rely on there.

If we cannot pinpoint which block is the problematic one, we can definitely break after the warning is present once yes! I would maybe suggest changing the phrasing a bit to signal that at least one block is faulty but there may be others.

```suggestion
			warns = append(warns, "at least one block's `snapshot_description` was ignored, as it needs `snapshot_volume` to be also set.")
```

Reply:
Alternatively, this could also conceivably be an error, as they happen early in the process (before the build even starts unless I'm mistaken), this would force users to address the issue before continuing.